test/plug/cast_test.exs

```diff
@@ -234,6 +234,23 @@ defmodule OpenApiSpex.Plug.CastTest do
                }
              }
     end
+
+    test "Optional param" do
+      conn =
+        :post
+        |> Plug.Test.conn("/api/pets/1/adopt?status=adopted")
+        |> Plug.Conn.put_req_header("content-type", "application/json; charset=UTF-8")
+      |> Plug.Conn.put_req_header("x-user-id", "123456")
```

Comment:
Here and on line 248: can the indentation be adjusted so that the code is consistently aligned?